src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,38 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters = [];
+    private string $routeName = '';
+
+    /**
+     * @param string $routeName
+     * @param array  $missingParameters
```

Comment:
Please use `string[]` to be more precise about the supported type (and also add the corresponding `@return string[]` on the getter)

Reply:
Fixed !